semillas_backend/static/css/layout/_header.scss

```diff
@@ -0,0 +1,98 @@
+// -----------------------------------------------------------------------------
+// This file contains all styles related to the header of the site/application.
+// -----------------------------------------------------------------------------
+.page-head--masthead .site-nav {
+  position: absolute;
+  width: 100%;
+  top: 0;
+}
+
+  .page-head--masthead {
+    background-position: center center;
+    -webkit-background-size: cover;
+       -moz-background-size: cover;
+            background-size: cover;
+    color: $backgroundColor;
+    box-shadow: 0 0 10px rgba(0,0,0,0.1) inset;
+  }
+
+    @media screen and (min-width: $bpMedium) {
+      .page-head--masthead {
+          background-image: url(/static/images/masthead-medium.jpg);
+      }
+    }
+
+    @media screen and (min-width: $bpLarge) {
+      .page-head--masthead {
+          background-image: url(/static/images/masthead-large.jpg);
```

Comment:
For this I would try to use relative paths... maybe... 

Check out how alpha stores its statics: 
The browser is looking for it (failing) here:
https://s3.amazonaws.com/static/images/masthead-large.jpg

While it is stored here:
https://s3.amazonaws.com/semillaalpha/images/masthead-large.jpg